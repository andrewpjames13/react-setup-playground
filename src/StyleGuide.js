import React, { Component } from 'react';

class StyleGuide extends Component {
  render() {
    return (
      <h2>AngelMD StyleGuide</h2>
    );
  }
}

export default StyleGuide;
