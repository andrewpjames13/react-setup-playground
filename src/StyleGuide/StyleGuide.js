import React, { Component } from 'react';
import ColorBlocks from './components/ColorBlocks';
import grid from 'Styles/grid.scss';

class StyleGuide extends Component {
  render() {
    return (
      <div className={`${grid.row} ${grid.maxWidth}`}>
        <div className={grid.row}>
          <h2>AngelMD StyleGuide</h2>
        </div>
        <div className={grid.row}>
          <div className={`${grid.xs100} ${grid.sm25}`}>
            <ColorBlocks color="primary" />
            <ColorBlocks color="primaryLight" />
          </div>
          <div className={`${grid.xs100} ${grid.sm25}`}>
            <ColorBlocks color="secondary" />
            <ColorBlocks color="secondaryMedium" />
            <ColorBlocks color="secondaryLight" />
          </div>
        </div>
      </div>
    );
  }
}

export default StyleGuide;
