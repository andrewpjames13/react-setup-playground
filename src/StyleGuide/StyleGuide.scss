@import "~Shared/styles/variables";

.header {
  color: $error;
}
