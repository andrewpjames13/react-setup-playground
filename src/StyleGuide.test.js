import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import StyleGuide from './StyleGuide';

it('renders without crashing', () => {
  const wrapper = shallow(<StyleGuide />);
  // const welcome = <h2>AngelMD StyleGuide</h2>;

  expect(wrapper.contains(<h2>AngelMD StyleGuide</h2>)).toEqual(true);
});

test('snapshot', () => {
  const tree = renderer.create(
    <StyleGuide/>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
