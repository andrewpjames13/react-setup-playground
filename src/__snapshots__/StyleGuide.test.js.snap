exports[`test snapshot 1`] = `
<h2>
  AngelMD StyleGuide
</h2>
`;
